modules/Microsoft.Sql/servers/encryptionProtector/readme.md

```diff
@@ -13,7 +13,7 @@ This module deploys an Sql Servers Encryption Protector.
 
 | Resource Type | API Version |
 | :-- | :-- |
-| `Microsoft.Sql/servers/encryptionProtector` | [2021-11-01](https://learn.microsoft.com/en-us/azure/templates/Microsoft.Sql/2021-11-01/servers/encryptionProtector) |
+| `Microsoft.Sql/servers/encryptionProtector` | [2022-08-01-preview](https://learn.microsoft.com/en-us/azure/templates/Microsoft.Sql/servers/encryptionProtector) |
```

Comment:
Duplicate of #3084

Reply:
Thanks! Closing as duplicate and approved the PR